Core/Src/main.c

```diff
@@ -65,7 +68,8 @@ void SystemClock_Config(void);
 int main(void)
 {
   /* USER CODE BEGIN 1 */
-
+	uint8_t spi_buf[2]; //buffer to hold digital potentiometer 8-bit values
```

Comment:
recommend you make the receive buffer be of length 3.

Reply:
Addressed :)